listenbrainz_spark/recommendations/recommend.py

```diff
@@ -290,7 +291,7 @@ def get_message_for_inactive_users(messages, active_users, users):
     return messages
 
 
-def get_recommendations_for_all(params, users):
+def get_recommendations_for_all(params, users, ti):
```

Comment:
should add an explanation in the docstring here of what ti is. Also, we generally use ts for timestamps